azure-pipelines.yml

```diff
@@ -46,9 +46,21 @@ jobs:
     displayName: 'Generate certificates'
 
   - task: PowerShell@2
-    displayName: 'Run NUKE Build'
+    displayName: 'Build images'
     inputs:
       targetType: 'filePath'
       filePath: build.ps1
       arguments: > 
           Xc
+  
+  - powershell: |
+      $gitTag=(git describe --exact-match)
```

Comment:
Deze powershell in een apart script zetten zodat dit lokaal ook kan worden getest als dat nodig is